ng build/serve fails with "Error: No errors" message

### OS?
`Ubuntu 16.10 
`
### Versions.
 ```
 __ _  _ __    __ _  _   _ | |  __ _  _ __         ___ | |(_)
 / _` || '_ \  / _` || | | || | / _` || '__|_____  / __|| || |
| (_| || | | || (_| || |_| || || (_| || |  |_____|| (__ | || |
 \__,_||_| |_| \__, | \__,_||_| \__,_||_|          \___||_||_|
               |___/
angular-cli: 1.0.0-beta.26
node: 7.4.0
os: linux x64
@angular/common: 2.4.5
@angular/compiler: 2.4.5
@angular/core: 2.4.5
@angular/forms: 2.4.5
@angular/http: 2.4.5
@angular/platform-browser: 2.4.5
@angular/platform-browser-dynamic: 2.4.5
@angular/router: 3.4.5
@angular/compiler-cli: 2.4.5
```



### Repro steps.
New app created using => 
`ng new PROJECT
`
### The log given by the failure.
 ng serve

```
No errors
Error: No errors
    at validate (/home/carlos/empresa.com.br/PROJECT/node_modules/extract-text-webpack-plugin/schema/validator.js:10:9)
    at Function.ExtractTextPlugin.extract (/home/carlos/empresa.com.br/PROJECT/node_modules/extract-text-webpack-plugin/index.js:188:3)
    at /home/carlos/empresa.com.br/PROJECT/node_modules/angular-cli/models/webpack-build-styles.js:79:83
    at Array.map (native)
    at Object.getWebpackStylesConfig (/home/carlos/empresa.com.br/PROJECT/node_modules/angular-cli/models/webpack-build-styles.js:76:43)
    at new NgCliWebpackConfig (/home/carlos/empresa.com.br/PROJECT/node_modules/angular-cli/models/webpack-config.js:44:51)
    at Class.run (/home/carlos/empresa.com.br/PROJECT/node_modules/angular-cli/tasks/serve-webpack.js:23:22)
    at /home/carlos/empresa.com.br/PROJECT/node_modules/angular-cli/commands/serve.run.js:37:22
    at process._tickCallback (internal/process/next_tick.js:103:7)

```

This is just a new ubuntu and a new node npm and angular-cli install 

`carlos@carlos-Z97M-D3H:~/empresa.com.br/PROJECT(master○) » npm start                                                             1 ↵

> project@0.0.0 start /home/carlos/empresa.com.br/PROJECT
> ng serve

```
No errors
Error: No errors
    at validate (/home/carlos/empresa.com.br/PROJECT/node_modules/extract-text-webpack-plugin/schema/validator.js:10:9)
    at Function.ExtractTextPlugin.extract (/home/carlos/empresa.com.br/PROJECT/node_modules/extract-text-webpack-plugin/index.js:188:3)
    at /home/carlos/empresa.com.br/PROJECT/node_modules/angular-cli/models/webpack-build-styles.js:79:83
    at Array.map (native)
    at Object.getWebpackStylesConfig (/home/carlos/empresa.com.br/PROJECT/node_modules/angular-cli/models/webpack-build-styles.js:76:43)
    at new NgCliWebpackConfig (/home/carlos/empresa.com.br/PROJECT/node_modules/angular-cli/models/webpack-config.js:44:51)
    at Class.run (/home/carlos/empresa.com.br/PROJECT/node_modules/angular-cli/tasks/serve-webpack.js:23:22)
    at /home/carlos/empresa.com.br/PROJECT/node_modules/angular-cli/commands/serve.run.js:37:22
    at process._tickCallback (internal/process/next_tick.js:103:7)

npm ERR! Linux 4.8.0-34-generic
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v7.4.0
npm ERR! npm  v4.0.5
npm ERR! code ELIFECYCLE
npm ERR! project@0.0.0 start: `ng serve`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the project@0.0.0 start script 'ng serve'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the project package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ng serve
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs project
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls project
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/carlos/empresa/PROJECT/npm-debug.log
```
`

```
carlos@carlos-Z97M-D3H:~/empresa.com.br/PROJECT(master⚡) » npm ls extract-text-webpack-plugin
project@0.0.0 /home/carlos/empresa.com.br/PROJECT
└── (empty)

npm ERR! code 1

```

--- a/package.json
+++ b/package.json
@@ -55,7 +55,7 @@
     "ember-cli-string-utils": "^1.0.0",
     "enhanced-resolve": "^2.3.0",
     "exists-sync": "0.0.3",
-    "extract-text-webpack-plugin": "^2.0.0-beta.5",
+    "extract-text-webpack-plugin": "^2.0.0-rc.1",
     "file-loader": "^0.8.5",
     "findup": "0.1.5",
     "fs-extra": "^0.30.0",

--- a/packages/angular-cli/models/webpack-configs/styles.ts
+++ b/packages/angular-cli/models/webpack-configs/styles.ts
@@ -92,7 +92,6 @@ export function getStylesConfig(wco: WebpackConfigOptions) {
   if (globalStylePaths.length > 0) {
     rules.push(...baseRules.map(({test, loaders}) => ({
       include: globalStylePaths, test, loaders: ExtractTextPlugin.extract({
-        remove: false,
         loader: [
           // css-loader doesn't support webpack.LoaderOptionsPlugin properly,
           // so we need to add options in its query

--- a/packages/angular-cli/package.json
+++ b/packages/angular-cli/package.json
@@ -44,7 +44,7 @@
     "ember-cli-normalize-entity-name": "^1.0.0",
     "ember-cli-string-utils": "^1.0.0",
     "exists-sync": "0.0.3",
-    "extract-text-webpack-plugin": "^2.0.0-beta.4",
+    "extract-text-webpack-plugin": "^2.0.0-rc.1",
     "file-loader": "^0.8.5",
     "findup": "0.1.5",
     "fs-extra": "^0.30.0",

